### DISCUSSION TITLE
Unable to Execute Tests with Edge Browser via Jenkins

### DISCUSSION THREAD

**Discussion by nuweq** [seleniumbase/SeleniumBase#2593] [Q&A] [answered]
I encountered an error while attempting to run my test using Jenkins with the Edge browser. It works fine if executed from PyCharm. Could you assist me in resolving this issue?
![image](https://github.com/seleniumbase/SeleniumBase/assets/37559617/bd574f61-c1af-49f6-b9f8-fffa84582cf9)

**Comment by mdmintz** [comment, marked as answer]
If you're using the `Driver()` format in a headless environment such as Linux, you'll need to wrap your browser with a virtual display separately.

See https://github.com/mdmintz/sbVirtualDisplay and https://github.com/seleniumbase/SeleniumBase/issues/2352#issuecomment-1848409823.

Be sure to include better debugging details in the future, such as the SeleniumBase version and the line of code you used to spin up your browser.
